packages/flutter_tools/lib/src/commands/update_packages.dart

```diff
@@ -107,6 +115,25 @@ class UpdatePackagesCommand extends FlutterCommand {
     final bool isPrintPaths = argResults['paths'];
     final bool isPrintTransitiveClosure = argResults['transitive-closure'];
     final bool isVerifyOnly = argResults['verify-only'];
+    final bool isConsumerOnly = argResults['consumer-only'];
+
+    // "consumer" packages are those our users are likely to rely on, and represents
```

Comment:
"those our users are likely to rely on" sounds wishy washy - how about "those that come from packages that constitute our public API (e.g. flutter, flutter_test, flutter_driver, flutter_localizations)"